test/DacpacTool.Tests/DacpacTool.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
```

Comment:
nit: this one's also out of order. Totally fine to ignore these if there are no other changes to be made :slightly_smiling_face: